ignite cli does not seems to work

ignite cli does not seems to work

Get the below error on 
$ node -v
v4.2.2
$ npm -v
3.5.3
$npm -g list --depth=0
├── babel-eslint@4.1.6
├── bower@1.7.7
├── generator-react-native-ignite@0.0.4
├── generator-webapp@2.0.0
├── gulp@3.9.0
├── gulp-cli@1.2.1
├── npm@3.5.3
├── react-native-cli@0.2.0
├── react-native-ignite@0.1.10
└── yo@1.7.0

$ ignite new react-native-aics
🔥 Setting react-native-aics on FIRE! 🔥
generate app -> react-native-aics ☕️  This will take a while ☕️
events.js:141
      throw er; // Unhandled 'error' event
      ^

ReferenceError: colors is not defined


- [x] Issue does not already exist
- Identify exactly what aspect your issue is about
  - [ ] ignite base applicattion
  - [x] ignite the cli
  - [ ] ignite generated code

yes the reference error color issue is gone - but it seems there is another error now when we run 
`ignite new app` 
`Error: Cannot find module '../validation' `
Guess this is already raised as an issue
#75

--- a/ignite-generator/package.json
+++ b/ignite-generator/package.json
@@ -6,7 +6,8 @@
     "app",
     "screen",
     "component",
-    "container"
+    "container",
+    "validation.js"
   ],
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",

